asv.conf.json

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
Config file for asv. See https://asv.readthedocs.io/en/latest/asv.conf.json.html